packages/frontend/src/utils/wallet.ts

```diff
@@ -133,10 +158,9 @@ export default class Wallet {
             removeAllAccountsPrivateKey();
         }
 
-        this.inMemorySigner = new nearApiJs.InMemorySigner(this.keyStore);
-        this.inMemorySignerBasic = new nearApiJs.InMemorySigner(this.keyStore);
+        this.signerIgnoringLedger = new nearApiJs.InMemorySigner(this.keyStore);
 
-        const inMemorySigner = this.inMemorySigner;
+        const signerIgnoringLedger = this.signerIgnoringLedger;
         const wallet = this;
 
         this.signer = {
```

Comment:
Could we also add types to the return values of this objects functions ?

Reply:
we can always come back to enhance this file in the future, but at least use a .ts extension with any is always better than use .js